pre-build-buildpack

```diff
@@ -10,11 +10,12 @@ DIR=/app
 COMMAND=$(cat <<EOF
 export DEBIAN_FRONTEND=noninteractive
 if [ -f $DIR/apt-repositories ]; then
+    apt-get install -y language-pack-en-base
```

Comment:
Is this package really required? To me it would be required for the `en_US.UTF-8` locale, but not for `C.UTF-8 `.

Reply:
Seems like it isn't: https://github.com/oerdnj/deb.sury.org/issues/56#issuecomment-191748654